infra/macos/2-common-box/README.md

```diff
@@ -0,0 +1,60 @@
+# Host setup
+
+The host machine for this step needs three things:
+
+1. The [Vagrant base box](../1-create-box/README.md) for macOS.
+2. VirtualBox (with the extension).
+3. Vagrant.
+
+As all three are already covered by the `macinbox` step, the easiest approach
+is probably to just run this on the same box.
+
+You can refer to the instructions in [this README](../1-create-box/README.md),
+or, for a more automated setup, use the following set of commands (which will
+require a sudoer password at some points):
+
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)" < /dev/null
+brew cask install virtualbox
+brew cask install vagrant
+cd $(mktemp -d)
+wget https://download.virtualbox.org/virtualbox/6.1.4/Oracle_VM_VirtualBox_Extension_Pack-6.1.4.vbox-extpack
```

Comment:
Any reason not to move to 6.1.6? What does cask install by default?

Reply:
Oh I just did not touch that part. I'll update.